gamemode/modules/base/sh_util.lua

```diff
@@ -34,13 +34,16 @@ function DarkRP.formatMoney(n)
     if n >= 1e14 then return attachCurrency(tostring(n)) end
     if n <= -1e14 then return "-" .. attachCurrency(tostring(math.abs(n))) end
 
+    local config = GAMEMODE.Config
+
     local negative = n < 0
 
     n = tostring(math.abs(n))
-    local dp = string.find(n, "%.") or #n + 1
+
+    local dp = string.find(n, ".", 1, true) or #n + 1
```

Comment:
Ah, this is a little clearer than the escaped dot. I had to look up what `'%.` meant again.